src/content/docs/release-notes/docs-release-notes/docs-10-11-2024.mdx

```diff
@@ -0,0 +1,67 @@
+---
+subject: "Docs"
+releaseDate: '2024-10-11'
+version: 'version: October 4-October 10, 2024'
+---
+
+### New docs
+
+* [No crash data appears in release build (Android)](/docs/mobile-monitoring/new-relic-mobile-android/troubleshoot/no-crash-data-appears) tells you what to do if you don't see data related to an application crash on the New Relic UI.
+* [How can I use WSL and CodeStream?](/docs/codestream/troubleshooting/using-wsl) provides step-by-step instructions on using Windows Subsystem for Linux (WSL) with CodeStream.
+* [User actions](/docs/browser/browser-monitoring/browser-pro-features/user-actions) is a new event type in browser monitoring that helps you understand user behavior with your web application.
+* [WithLlmCustomAttributes (Python agent API)](/docs/apm/agents/python-agent/python-agent-api/withllmcustomattributes-python-agent-api) describes the new context manager API, including its requirements, parameters, return values, and an example usage.
+
+### Minor changes
+
+* Added US and EU data center endpoints for the APM agent in [New Relic telemetry endpoints](/docs/new-relic-solutions/get-started/networks/#new-relic-endpoints).
+* Updated instructions on how to find CodeStream client-side logs when using [Visual Studio](/docs/codestream/troubleshooting/client-logs/#visual-studio).
+* Reworded [Notification options for muting rules](/docs/alerts/get-notified/muting-rules-suppress-notifications/#notify) for better clarity.
```

Comment:
...notification options (check capitalisation)

Reply:
Discussed capitalization with Chinmay - retaining as is